### PR TITLE
Run scripted with sbt-maven-resolver

### PR DESCRIPTION
**Pull request by eed3si9n**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ env:
     - SCRIPTED_TEST="scripted compiler-project/*""
     - SCRIPTED_TEST="scripted dependency-management/*1of2"
     - SCRIPTED_TEST="scripted dependency-management/*2of2"
+    - SCRIPTED_TEST="mavenResolverPluginTest:scripted dependency-management/*1of2"
```
Comment (jsuereth) [inline .travis.yml:19]: Sweet.  Can we add project/transitive-plugins here?

File: project/Scripted.scala [hunk 2]
```diff
@@ -55,14 +58,14 @@ object Scripted {
       (token(Space) ~> (PagedIds | testIdAsGroup)).* map (_.flatten)
     }
 
-  def doScripted(launcher: File, scriptedSbtClasspath: Seq[Attributed[File]], scriptedSbtInstance: ScalaInstance, sourcePath: File, args: Seq[String]) {
+  def doScripted(launcher: File, scriptedSbtClasspath: Seq[Attributed[File]], scriptedSbtInstance: ScalaInstance, sourcePath: File, args: Seq[String], prescripted: File => Unit) {
```
Comment (jsuereth) [inline project/Scripted.scala:61]: Do we need a BC version of this somewhere?  Hopefully not....

**Comment by jsuereth** [comment]
LGTM. One BC question, then mergeable (and I can add the other critical aether test).

File: project/Scripted.scala (hunk 2 above)
Comment (eed3si9n) [inline project/Scripted.scala:61]: It's our own `project/*.scala` build code. No need for BC.

**Pull request merged**